lib/LaTeXML/Core/Document.pm

```diff
@@ -1970,7 +1970,7 @@ sub renameNode {
     my $key   = $attr->getName;
     my $value = $node->getAttribute($key);
     $id = $value if $key eq 'xml:id';    # Save to register after removal of old node.
-    $new->setAttribute($key, $value); }
+    $new->setAttribute($key, $value) if $model->canHaveAttribute($newname, $key); }
```

Comment:
should we emit an Info message if an attribute gets dropped here? I wonder if it may aid some future debugging session.

Reply:
hmm, yeah maybe, but probably just a lot of noise?